### PR TITLE
Link to Planning Poker community project

### PR DESCRIPTION
**Pull request by duracellko**
Add link to Planning Poker application implemented using Blazor. Link is added to community.md.

### CODE REVIEW

**Comment by guardrex** [comment]
Should "Planning Poker" be linked to :point_right: https://www.planningpoker.com/

**Comment by duracellko** [comment]
> Should "Planning Poker" be linked to  https://www.planningpoker.com/

I think rather https://en.m.wikipedia.org/wiki/PlanningPoker or https://www.mountaingoatsoftware.com/agile/planning-poker. That explains what Planning Poker is.

**Comment by guardrex** [comment]
Ok ... I was wondering if it was a trademark, thus perhaps associated with that website. @danroth27 will come on to take a look and let us know which one he prefers.

**Comment by guardrex** [comment]
"Planning Poker" is a registered trademark of Mountain Goat Software LLC (associated with the website planningpoker.com), but their trademark industry classifications may allow you to use the name in this capacity. For the Community page, it would be vastly safer for Microsoft and the .NET Foundation if we note the trademark and link to the planningpoker.com website. Otherwise for the PR as-is, I suggest asking CELA to take a look before we accept this. Dan will know what to do.

The industry classifications are listed at the TESS website if you want to research them further:

http://tess2.uspto.gov/bin/showfield?f=doc&state=4805:eeg9tq.2.1

**Comment by duracellko** [comment]
I didn't know "Planning Poker" is registered trademark. I added link to the web.

File: community.md [hunk 1]
```diff
@@ -80,6 +80,7 @@ Below is a list of projects and resources from the Blazor community (links in ea
 * [Data Driven Layout](https://github.com/hutchcodes/Blazor.DataDrivenLayout) ([Blog post](https://hutchcodes.net/2018/09/data-driven-layout-in-razor-components/)) &ndash; Two ways of driving the layout based on data in the page.
 * [Do](https://github.com/jamie-lord/do) ([Demo](https://do.lord.technology)) &ndash; Todo.txt compatible todo app.
 * [Money](https://github.com/maraf/Money) ([Demo](https://money.neptuo.com)) &ndash; A money manager implemented using CQRS+ES.
+* [Planning Poker](https://github.com/duracellko/planningpoker4azure) ([Demo](http://planningpoker.duracellko.net)) &ndash; An app to play [Planning Poker](https://www.planningpoker.com) for distributed teams. The app is implemented using Blazor and shows how to switch between client-side and server-side mode with a configuration change.
```
Comment (guardrex) [inline community.md]:
```suggestion
* [Planning Poker for Azure](https://github.com/duracellko/planningpoker4azure) ([Demo](http://planningpoker.duracellko.net)) &ndash; An app to play [Planning Poker&trade;](https://www.planningpoker.com) for distributed teams. The app is implemented using Blazor and shows how to switch between client-side and server-side mode with a configuration change. *Planning Poker* is a registered trademark of [Mountain Goat Software, LLC](https://www.mountaingoatsoftware.com/).
```

**Comment by guardrex** [comment]
I went ahead with the mods that I suggested. Dan will let us know on review where we stand with the trademarked name and if my mods are sufficient.

**Pull request merged**